articles/implementing-mvvm-architecture-in-android-using-kotlin/index.md

```diff
@@ -0,0 +1,71 @@
+
+Let's dive in!
+
+
+### Prerequisites
+1. Android studio
+2. You must be familiar with [Kotlin](https://developer.android.com/kotlin/campaign/learn?gclid=Cj0KCQiA7qP9BRCLARIsABDaZzh1wodOJn7w8kKTtWq8yNFlx9xoqzEE_cU2KkCO2Ecdyyr2frGOVjQaAlSuEALw_wcB&gclsrc=aw.ds)
+3. Install [lifecycle](https://developer.android.com/jetpack/androidx/releases/lifecycle) dependencies
+4. Download the start code from [here](https://github.com/WanjaMIKE/MVVMExample)
+
+#### The goal of the tutorial
+By the end of this tutorial, you will create an app that takes input and displays it on a recycler view. A screenshot of the app is shown below.
+
+### Step 1 – Launching Android Studio.
+Launch Android Studio and create a new project, as shown below. Make sure that you select Kotlin as your preferred programming language. If you do not have Android Studio, you can install it from [here](https://developer.android.com/studio?gclid=Cj0KCQiA7qP9BRCLARIsABDaZzieBJWjBnokDdH6b0gQchoqudRXNohAGp_noSqALLuSlYuwA6EB5T4aAntwEALw_wcB&gclsrc=aw.ds)
+
+![getting started](/engineering-education/implementing-mvvm-architecture-in-android-using-kotlin/getting-started.png)
+
+### Step 2 – Creating the model.
+Create the app model. This is also referred to as the data class. To avoid confusion, create a package named model inside the java folder. Then, create a data class named Blog in the model package as shown below. For simplicity, the data class will only have one variable (title). In Kotlin, there is no need to use the get and set keywords. These functionalities are added to the data class automatically.
+
+![model](/engineering-education/implementing-mvvm-architecture-in-android-using-kotlin/model.png)
+
+### Step 3 – Creating the view.
+The view is what the user sees on the screen. The UI, therefore, needs to be well structured to minimize confusion and dissatisfaction.
+
+Open `activity_main.xml` file and change the layout from constraint to linear layout. Ensure that the orientation is set to vertical. This ensures that the UI components such as buttons and text views are arranged vertically. For our app, the primary widgets are `Edittext`, `Button`, and a `RecyclerView`. Ensure that all these widgets have IDs since they will be required at a later stage. The layout design and code are shown in the image below. The section with a list showing items 0 to 9 is the `RecyclerView`. Whatever we type in the app will be displayed in that section once the submit button is clicked.
```

Comment:
Replace "This ensures that the UI components... " with "This arranges UI components vertically in the Layout.". It easier to read.